django_select2/forms.py

```diff
@@ -108,6 +108,21 @@ def _get_media(self):
 
     media = property(_get_media)
 
+    def label_from_instance(self, obj):
```

Comment:
I'm missing the implementation for the `light` version. This should be really consistent.